src/uv.cc

```diff
@@ -59,6 +59,16 @@ using v8::ReadOnly;
 using v8::String;
 using v8::Value;
 
+void GetErrMessage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  int err;
+  if (!args[0]->Int32Value(env->context()).To(&err)) return;
+  CHECK_LT(err, 0);
+  char message[50];
```

Comment:
Why 50? Can you add a comment?

Reply:
Same as the name getter

https://github.com/juanarbol/node/blob/cf6d05950d991b0cbeb06aa10d5bcf79c485c81f/src/uv.cc#L86

Reply:
Can you add a comment?